open-broker-common/src/main/java/de/fraunhofer/iais/eis/ids/broker/core/common/persistence/ResourceMessageHandler.java

```diff
@@ -52,14 +59,25 @@ public ResourceMessageHandler(ResourceStatusHandler resourceStatusHandler, Infra
      */
     @Override
     public DefaultSuccessMAP handleValidated(ResourceMAP messageAndPayload) throws RejectMessageException {
+		XMLGregorianCalendar issueAt = messageAndPayload.getMessage().getIssued();
+
         ResourceNotificationMessage msg = (ResourceNotificationMessage) messageAndPayload.getMessage();
         MessageLogger.logMessage(messageAndPayload, true, "affectedResource");
         URI rewrittenUri = null;
         try {
             if (msg instanceof ResourceUpdateMessage) {
                 //ResourceUpdateMessages have the affected Resource in their payload
-                if (msg.getAffectedResource() != null && messageAndPayload.getPayload().isPresent()) {
 
+
+                if (msg.getAffectedResource() != null && messageAndPayload.getPayload().isPresent()) {
+                	
+                	 logger.info(issueAt.getYear()+":"+issueAt.getMonth()+ ":"+ issueAt.getDay()+":"+issueAt.getHour()+":"+ issueAt.getMinute());
+                     logger.info("Component ID :"+ infrastructureComponent.getId());
```

Comment:
This will be difficult to maintain. We will need to log a ton of messages, so it would be better to have one function to print log entries for a generic message. That makes the code re-usable for other messages than ResourceUpdateMessages